### ISSUE TITLE
Getting an error with lib/armeabi-v7a/libjingle_peerconnection_so.so

### ISSUE THREAD

**Issue by javiersuazo** [oney/react-native-webrtc#149]
I'm getting this error:

```
Execution failed for task ':WebRTCModule:transformNative_libsWithSyncJniLibsForRelease'.
> java.io.FileNotFoundException: /Users/javiersuazo/AA/cyclops-mobile/node_modules/react-native-webrtc/android/build/intermediates/bundles/release/jni/lib/armeabi-v7a/libjingle_peerconnection_so.so (No such file or directory 
```

When I try to build my project in android. I'm using the version 0.13.0

**Comment by zxcpoiu** [comment]
Hi @javiersuazo 

would you mind share what's the cause and how you got through it?
it may helps others and can be added to installation guide
thanks!

**Comment by gmahomarf** [comment]
@zxcpoiu I can answer that. It's an issue with the android gradle plugin since version 1.5 of the plugin. It was finally fixed in version 2.2, so we just needed to update. See https://code.google.com/p/android/issues/detail?id=193063 for details.
